include/librealsense2/h/rs_option.h

```diff
@@ -340,6 +340,27 @@ extern "C" {
     */
     const char* rs2_get_option_value_description(const rs2_options* options, rs2_option option, float value, rs2_error ** error);
 
+    /**
+    * Sets a callback in case an option in this options container value is updated
+    * Will create a thread that will periodically check the options in the container for updates
```

Comment:
Add `The update period is determined by the context's 'options-update-interval' setting`

Reply:
Added